### PR TITLE
[full-ci] feat: add pagination

### PR DESCRIPTION
**Pull request by LukasHirt**
## Changes
- Add pagination
- Add `page` param to routes with files lists
- Setup integration tests
- **BREAKING** move all unit tests into `**/unit/` folders

## Integration tests
- Resources get updated when user navigates to a new page via pagination in Personal files list
- Resources get updated when user navigates to a new page via pagination in Favorites files list
- Resources get updated when user navigates to a new page via pagination in LocationPicker files list
- Resources get updated when user navigates to a new page via pagination in PublicFiles files list
- Resources get updated when user navigates to a new page via pagination in SharedViaLink files
- Resources get updated when user navigates to a new page via pagination in SharedWithMe files list
- Resources get updated when user navigates to a new page via pagination in SharedWithOthers files list
- Resources get updated when user navigates to a new page via pagination in Trashbin files list
- Resources get updated when user navigates to Personal directly via URL
- Resources get updated when user navigates to Favorites directly via URL
- Resources get updated when user navigates to LocationPicker directly via URL
- Resources get updated when user navigates to PublicFiles directly via URL
- Resources get updated when user navigates to SharedViaLink directly via URL
- Resources get updated when user navigates to SharedWithMe directly via URL
- Resources get updated when user navigates to SharedWithOthers directly via URL
- Resources get updated when user navigates to Trashbin directly via URL

## Screenshot
Screenshot 2021-06-15 at 11 23 01

## Todo
- [x] reset current page on search / clearing search
- [x] webUIMarkdownEditor/markdownFile.feature:26 @LukasHirt 
- [x] webUIFiles/breadcrumb.feature:20 @LukasHirt

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIOCISFiles2</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCISSharingInternalUsers2</strong> failed. The build is cancelled...</summary>

**Comment by LukasHirt** [comment]
Something for separate PR - merge unit and integration tests coverage to get the real coverage result

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16258/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16258/

<p>


### webUIFilesDetails-fileDetails-feature-116.png
  

### webUIFilesDetails-fileDetails-feature-66.png
  

### webUIFilesDetails-fileDetails-feature-82.png
  

### webUIFilesDetails-fileDetails-feature-99.png
  

### webUIFilesList-fileList-feature-46.png
  


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16261/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16261/

<p>


### webUIFilesDetails-fileDetails-feature-116.png
  

### webUIFilesDetails-fileDetails-feature-66.png
  

### webUIFilesDetails-fileDetails-feature-82.png
  

### webUIFilesDetails-fileDetails-feature-99.png
  

### webUIFilesList-fileList-feature-46.png
  


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. The build is cancelled...</summary>

**Review by pascalwengerter — COMMENTED** [review]
Works like a charm, I'm wondering if we want to go with the left- and right-alignment of the pagination and files/folder count sections?

I'd vote in favor of having them centered for a more organized look. Also, when no pagination is shown it looks off having the file/folder count aligned to the right with vast emptiness left & center

Screenshot 2021-06-15 at 10 52 19

**Comment by pascalwengerter** [comment]
> Works like a charm, I'm wondering if we want to go with the left- and right-alignment of the pagination and files/folder count sections?
> 
> I'd vote in favor of having them centered for a more organized look. Also, when no pagination is shown it looks off having the file/folder count aligned to the right with vast emptiness left & center
> 
> Screenshot 2021-06-15 at 10 52 19

Also, from an a11y standpoint we should also maybe give context on whether the displayed file/folder count&size is defined by the current page or all files in the accommodating folder?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFilePermission</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>OCIS-Trashbin-Upload-Move-Journey</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16376/

<p>


### webUIFiles-breadcrumb-feature-20.png
  


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. The build is cancelled...</summary>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIOCISFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16380/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16380/

<p>


### webUIFiles-breadcrumb-feature-20.png
  


</p></details>

File: tests/helpers/toKebabCase.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function(string) {
```
Comment (kulmann) [inline tests/helpers/toKebabCase.js:1]: Could be replace with `kebabCase` from [lodash](https://lodash.com/docs/4.17.15#kebabCase)

**Review by kulmann — CHANGES_REQUESTED** [review]

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16384/

<p>


### webUIFiles-breadcrumb-feature-20.png
  


</p></details>

File: packages/web-app-files/src/components/FilesListFooterInfo.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<template functional>
+  <div
+    :ref="data.ref"
+    v-bind="data.attrs"
+    class="uk-text-nowrap oc-text-muted uk-text-center"
+    :class="[data.staticClass, data.class]"
+  >
+    <!--
+      TODO: merge all strings here to one interpollated translation
```
[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/components/FilesListFooterInfo.vue:9]: Thinking about it, the relevant information here is not about pagination, but about the current folder in general. The fact that it's paginated is IMO not relevant for the info text. If we want to show how many items are currently displayed, that's something for extending the pagination even more.

[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/components/FilesListFooterInfo.vue]: Changed it to this, what do you think?

Screenshot 2021-06-16 at 23 49 29

[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/components/FilesListFooterInfo.vue]: Feel free to revert the commit and do something else

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>IntegrationApp1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16388/

<p>


### webUIFavorites-favoritesFile-feature-14.png
  

### webUIFavorites-unfavoriteFile-feature-13.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16388/

<p>


### webUIFilesDetails-fileDetails-feature-116.png
  

### webUIFilesDetails-fileDetails-feature-66.png
  

### webUIFilesDetails-fileDetails-feature-82.png
  

### webUIFilesDetails-fileDetails-feature-99.png
  

### webUIFilesList-sort-feature-31.png
  

### webUIFilesList-sort-feature-51.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16388/

<p>


### webUIFavorites-favoritesFile-feature-131.png
  

### webUIFavorites-favoritesFile-feature-14.png
  

### webUIFavorites-favoritesFile-feature-30.png
  

### webUIFavorites-favoritesFile-feature-65.png
  

### webUIFavorites-favoritesFile-feature-74.png
  

### webUIFavorites-unfavoriteFile-feature-13.png
  

### webUIFavorites-unfavoriteFile-feature-34.png
  

### webUIFiles-breadcrumb-feature-20.png
  


</p></details>

File: packages/web-app-files/src/store/getters.js [hunk 3]
```diff
@@ -29,12 +29,16 @@ export default {
...
     const direction = state.fileSortDirectionDesc ? 'desc' : 'asc'
-    return [].concat(files).sort(fileSortFunctions[state.fileSortField][direction])
+    const currentPageIndex = (state.currentPage - 1) * state.filesPageLimit
+
+    return []
+      .concat(files)
+      .sort(fileSortFunctions[state.fileSortField][direction])
+      .splice(currentPageIndex, state.filesPageLimit)
   },
+  paginationLength: state => Math.ceil(state.files.length / state.filesPageLimit),
```
[the remark this answers is not in the document]
Comment (kulmann) [reply packages/web-app-files/src/store/getters.js]: Renamed to `pages`. Please revert if you don't agree

File: tests/helpers/toKebabCase.js (hunk 1 above)
Comment (kulmann) [reply tests/helpers/toKebabCase.js:1]: Needs a decision first if we want to go with lodash / lodash-es / whatever. Keep it like it is for now.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>IntegrationApp1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16421/

<p>


### webUIFavorites-favoritesFile-feature-14.png
  

### webUIFavorites-unfavoriteFile-feature-13.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16421/

<p>


### webUIFavorites-favoritesFile-feature-131.png
  

### webUIFavorites-favoritesFile-feature-14.png
  

### webUIFavorites-favoritesFile-feature-30.png
  

### webUIFavorites-favoritesFile-feature-65.png
  

### webUIFavorites-favoritesFile-feature-74.png
  

### webUIFavorites-unfavoriteFile-feature-13.png
  

### webUIFavorites-unfavoriteFile-feature-34.png
  

### webUIFiles-breadcrumb-feature-20.png
  

### webUIMarkdownEditor-markdownFile-feature-26.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16443/

<p>


### webUIFiles-breadcrumb-feature-20.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16448/

<p>


### webUISharingInternalGroupsSharingIndicator-shareWithGroups-feature-136.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-124.png
  


</p></details>

**Review by kulmann — APPROVED** [review]
Awesome progress with the integration tests! Very cool pagination. LGTM 🥳 🚀 😍

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingInternalGroupsSharingIndicator-shareWithGroups-feature-136.png
  

### webUISharingInternalGroupsToRootSharingIndicator-shareWithGroups-feature-124.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersBlacklisted</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-26.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-44.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-62.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-80.png
  

### webUISharingInternalUsersBlacklisted-shareWithUsers-feature-92.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-17.png
  

### webUISharingInternalUsersSharingIndicator-shareWithUsers-feature-58.png
  

### webUISharingInternalUsersToRootSharingIndicator-shareWithUsers-feature-118.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingAcceptSharesToRoot-acceptShares-feature-31.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move-Upload</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUIUpload-upload-feature-89.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFilePermission</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers-feature-60.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers-feature-61.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers-feature-62.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers-feature-63.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers-feature-64.png
  

### webUISharingFilePermissionMultipleUsers-shareFileWithMultipleUsers-feature-65.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup-feature-61.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup-feature-62.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup-feature-63.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup-feature-64.png
  

### webUISharingFilePermissionsGroups-sharePermissionsGroup-feature-65.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderAdvancedPermissionMU</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-60.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-61.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-62.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-63.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-64.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-65.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-66.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-67.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-68.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-69.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-70.png
  

### webUISharingFolderAdvancedPermissionMultipleUsers-sharedFolderWithMultipleUsersAdvancedPermissions-feature-71.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderPermissions</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingFolderPermissionMultipleUsers-shareFolderWithMultipleUsers-feature-61.png
  

### webUISharingFolderPermissionMultipleUsers-shareFolderWithMultipleUsers-feature-62.png
  

### webUISharingFolderPermissionMultipleUsers-shareFolderWithMultipleUsers-feature-63.png
  

### webUISharingFolderPermissionMultipleUsers-shareFolderWithMultipleUsers-feature-64.png
  

### webUISharingFolderPermissionMultipleUsers-shareFolderWithMultipleUsers-feature-65.png
  

### webUISharingFolderPermissionMultipleUsers-shareFolderWithMultipleUsers-feature-66.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup-feature-61.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup-feature-62.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup-feature-63.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup-feature-64.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup-feature-65.png
  

### webUISharingFolderPermissionsGroups-sharePermissionsGroup-feature-66.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderAdvPermissionsGrp</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-61.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-62.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-63.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-64.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-65.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-66.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-67.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-68.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-69.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-70.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-71.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup-feature-72.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>IntegrationApp1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUIPrivateLinks-accessingPrivateLinks-feature-17.png
  

### webUIPrivateLinks-accessingPrivateLinks-feature-9.png
  

### webUIRestrictSharing-disableSharing-feature-16.png
  

### webUIRestrictSharing-restrictReSharing-feature-23.png
  

### webUIRestrictSharing-restrictReSharing-feature-42.png
  

### webUIRestrictSharing-restrictSharing-feature-31.png
  

### webUIRestrictSharing-restrictSharing-feature-40.png
  

### webUIRestrictSharing-restrictSharing-feature-56.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUIFilesDetails-fileDetails-feature-116.png
  

### webUIFilesDetails-fileDetails-feature-14.png
  

### webUIFilesDetails-fileDetails-feature-163.png
  

### webUIFilesDetails-fileDetails-feature-172.png
  

### webUIFilesDetails-fileDetails-feature-25.png
  

### webUIFilesDetails-fileDetails-feature-38.png
  

### webUIFilesDetails-fileDetails-feature-51.png
  

### webUIFilesDetails-fileDetails-feature-66.png
  

### webUIFilesDetails-fileDetails-feature-82.png
  

### webUIFilesDetails-fileDetails-feature-99.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>IntegrationApp2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingAutocompletion-shareAutocompletion-feature-32.png
  

### webUISharingAutocompletion-shareAutocompletion-feature-44.png
  

### webUISharingInternalUsers-shareWithUsers-feature-57.png
  

### webUISharingInternalUsers-shareWithUsers-feature-58.png
  

### webUISharingInternalUsers-shareWithUsers-feature-59.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-52.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-53.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-54.png
  

### webUISharingPublicBasic-publicLinkCreate-feature-11.png
  

### webUISharingPublicBasic-publicLinkCreate-feature-28.png
  

### webUISharingPublicDifferentRoles-shareByPublicLinkDifferentRoles-feature-33.png
  

### webUISharingPublicDifferentRoles-shareByPublicLinkDifferentRoles-feature-34.png
  

### webUISharingPublicDifferentRoles-shareByPublicLinkDifferentRoles-feature-35.png
  

### webUIUpload-upload-feature-89.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUILogin-adminBlocksUser-feature-20.png
  

### webUIPreview-imageMediaViewer-feature-140.png
  

### webUIPreview-imageMediaViewer-feature-158.png
  

### webUIPreview-imageMediaViewer-feature-83.png
  

### webUIPreview-imageMediaViewer-feature-90.png
  

### webUIPrivateLinks-accessingPrivateLinks-feature-17.png
  

### webUIPrivateLinks-accessingPrivateLinks-feature-25.png
  

### webUIPrivateLinks-accessingPrivateLinks-feature-9.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-100.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-109.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-31.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-32.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-33.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-34.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-37.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-44.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-54.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-74.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-75.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-78.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-86.png
  

### webUISharingInternalUsersToRootCollaborator-shareWithUsers-feature-93.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-11.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-25.png
  

### webUISharingInternalUsersToRootPreviews-shareWithUsers-feature-40.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-88.png
  

### webUISharingInternalUsersToRootShareWithPage-shareWithUsers-feature-96.png
  


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/16452/

<p>


### webUISharingInternalUsersToRoot-shareWithUsers-feature-103.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-129.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-142.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-161.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-183.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-197.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-212.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-255.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-256.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-257.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-275.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-52.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-53.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-54.png
  

### webUISharingInternalUsersToRoot-shareWithUsers-feature-74.png
  

### webUISharingInternalUsersToRootBlacklisted-shareWithUsers-feature-25.png
  

### webUISharingInternalUsersToRootBlacklisted-shareWithUsers-feature-41.png
  

### webUISharingInternalUsersToRootBlacklisted-shareWithUsers-feature-55.png
  

### webUISharingInternalUsersToRootBlacklisted-shareWithUsers-feature-69.png
  

### webUISharingInternalUsersToRootBlacklisted-shareWithUsers-feature-81.png
  


</p></details>

**Pull request merged**